src/message-store-sql.ts

```diff
@@ -28,6 +40,7 @@ export class MessageStoreSql implements MessageStore {
       .ifNotExists()
       .addColumn('tenant', 'text', (col) => col.notNull())
       .addColumn('messageCid', 'varchar(60)', (col) => col.notNull())
+      .addColumn('encodedData', 'text') // we optionally store encoded data if it is below a threshold
```

Comment:
I think for now, as DWNs are not production ready, we are probably safe for deleting everything and starting over. However, long term, the DWN team will really need to consider this as it will be a major issue for users wanting to upgrade.

Reply:
💯 totally agree... backward compatibility is extremely important here moving forward.